roles/maintenance_11_debian/tasks/main.yml

```
@@ -91,6 +91,24 @@
     update_cache: yes
   changed_when: no
 
+- <<: *task
+  vars:
+    taskid: 11-015
+    name: "GPG-Keys: Are there errors because of not existing GPG Keys?"
+  ansible.builtin.shell: apt-get update | grep -e 'Key is stored in legacy trusted.gpg keyring' -e 'GPG error:'  # noqa risky-shell-pipe
+  register: debian_apt_update_status
+  changed_when: no
+  check_mode: no
+  failed_when: "debian_apt_update_status.rc not in [0, 1]"
+
+- <<: *task
+  vars:
+    taskid: 11-015
+    name: "GPG-Keys: Are there errors because of not existing GPG Keys? | Report errors"
+  ansible.builtin.debug:
+    var: "debian_apt_update_status.stderr_lines"
+  changed_when: "debian_apt_update_status.stderr_lines | length > 0"
```

As my proposed change above redirects to stdout, this needs to be reflected here as well:

```suggestion
    var: "debian_apt_update_status.stdout_lines"
  changed_when: "debian_apt_update_status.stdout_lines | length > 0"
```